TRN-162-add-marks/index.php

```diff
@@ -0,0 +1,114 @@
+
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Form</title>
+        <style>
+            <?php
+                echo "td{padding:5px;}";
```

Comment:
Please explain these.

Reply:
it was added by mistake. I will change it.